resdk/resources/base.py

```diff
@@ -235,6 +235,29 @@ def permissions(self):
 
         return self._permissions
 
+    def _list_holders_with_perm(self, perm):
+        """Get users/group/public names that have ``perm`` perm."""
+        holders = []
+        self.permissions.fetch()
+
+        for item in self.permissions._permissions:  # pylint: disable=protected-access
+            if perm in item['permissions']:
+                holders.append(item.get('name', 'Public'))
+
+        return holders
+
+    def list_owners(self):
```

Comment:
This doesn't list users, but returns them, so more appropriate name would be `get_owners`. Considering the implementation of other similar cases in the ReSDK it may also make sense to make this a property.

Reply:
Have you thought about what makes more sense, adding this method to the `<object>` or to the `<object>.permissions`?

Reply:
My initial implementation was in `<object>.permissions`, yes. I changed my mind since it made more sense to say `Data.get_owners()` than `Data.permissions.get_owners()`. But the benefit of the `<object>.permissions` is that it clearly separates the "permission-handling" to `PermissionManager` class. What do you think?

If we decide to keep it in `<object>` i think making this a propery (`owners`, `editors`, ...) is a good choice. If we go with `<object>.permissions` i think it is more understandable if we use `get_owners()`.